tests/activemq5-unit-tests/src/test/java/org/apache/activemq/openwire/v1/ActiveMQBytesMessageTest.java

```diff
@@ -40,7 +40,6 @@ public Object createObject() throws Exception {
    @Override
    protected void populateObject(Object object) throws Exception {
       super.populateObject(object);
-      ActiveMQBytesMessage info = (ActiveMQBytesMessage) object;
```

Comment:
This removal then also renders the entire method superflous since all it does is call super.

I'd either leave it as it is, since these are originally generated files from ported tests (so the more we change them, the further from the originals they go)...or delete the methods instead of just the variable.

Same goes for the likely tens of other similar cases

Reply:
I reverted most of the changes in that module.